SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -4,14 +4,22 @@ Volume {
 
 	var <volume = 0.0, <lag = 0.1, <isMuted = false;
 
-	var <ampSynth, <numOutputChannels, defName, updateFunc, initFunc;
+	var <ampSynth, <numOutputChannels, <defName, updateFunc, initFunc;
 	var <>window;
+	var <group, baseAmpSynth;
 
 	*new { | server, startBus = 0, numChannels, min = -90, max = 6, persist = false |
 		^super.newCopyArgs(server ?? { Server.default }, startBus, numChannels, min, max, persist).init;
 	}
 
 	init {
+		// group nodeID should always be 2, so multiple clients can use it
+		group = Group.basicNew(server, server.nextPermNodeID);
```

Comment:
It may be that `server.nextPermNodeID` doesn't return the nodeID of 2. It depends on something rather brittle, namely on the call order for the server tree.

Reply:
I think the numbers are fixed and safe: 
I am making them correctly initially, and then keeping them forever:
Server:new creates its volume object in the init method, 
and Volume:init calls s.nextPermID twice, getting 2 and 3. 

But when the server boots, it resets nodeAllocator perm to 2, 
so the next two calls to it get 2 and 3 when they should be reserved. 

Mazbe the simplest fix: 
hardcode numbers 2 and 3 for volume group and synth,
and reset nodeAllocator perm to 4. 
yes?

Reply:
yes!